scheduler/pkg/agent/rproxy_grpc.go

```diff
@@ -235,22 +235,27 @@ func (rp *reverseGRPCProxy) ModelInfer(ctx context.Context, r *v2.ModelInferRequ
 		go rp.metrics.AddModelInferMetrics(externalModelName, internalModelName, metrics.MethodTypeGrpc, elapsedTime, codes.NotFound.String())
 		return nil, status.Error(codes.NotFound, fmt.Sprintf("Model %s not found (err: %s)", r.ModelName, err))
 	}
+
+	// Create an outgoing context for the proxy call to service from incoming context
+	outgoingCtx, requestId := rp.createOutgoingCtxWithRequestId(ctx)
```

Comment:
do we need to do something similar to `ModelMetadata` and `ModelReady` grpc calls?

Reply:
Probably need to decide if requestId is needed for these calls. Maybe a follow on or when we decide its needed.